app/controllers/pets_controller.rb

```diff
@@ -65,5 +78,13 @@ def pet_params
                   :adoptable)
   end
 
+  def empty_params
+    empty_params = []
+    empty_params << "Name" if params[:name] == ""
+    empty_params << "Age" if params[:age] == ""
+    empty_params << "Image" if params[:image] == ""
+    empty_params.join(", ")
+  end
```

Comment:
if any of the validated fields are not entered, add to a list to display in a flash message

Reply:
theres that join again